src/Shared/Microsoft.AspNetCore.Razor.Utilities.Shared/PooledObjects/PooledList`1.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
The filename for `PooledList<T>` seems to be incorrect.
Expected: `PooledList.cs`
Actual: ``PooledList`1.cs``

Reply:
This is a standard name for a generic type. It's the metadata name.

Reply:
The name is intentional to indicate that the type is generic. You'll find similar file names in both Roslyn and the .NET libraries.

Reply:
My bad :flushed: